src/components/Timer/Timer.tsx

```diff
@@ -54,6 +59,15 @@ export const Timer = (props: TimerProps) => {
     if (!previousPlayTimesUpState && playTimesUp) {
       timesUpSoundObject.on("end", () => setPlayTimesUp(false));
       playTimesUpSound();
+      if (isModerator) {
+        Toast.info(
+          <div>
+            <div>{t("Toast.moderatorResetReadyStates")}</div>
+            <Button onClick={() => API.resetAllReadyStates(boardId)}>{t("Toast.moderatorResetReadyStatesButton")}</Button>
```

Comment:
Like mentioned in our daily you could increase the margin top of the button here, so that there's at least a gap of 1 line in between the label and the button.

Reply:
Done. :-)